### ISSUE TITLE
Scene JSX: kill the `component` and `props`  prop, make the rendered component the real child of Scene

### ISSUE THREAD

**Issue by balazsbajorics** [concrete-utopia/utopia#297]
living paper link:
https://paper.dropbox.com/doc/Direct-Layout-Manipulation-Open-Ended-Problems--A34gaYYgK9CT~jz1KnGEK3uiAg-XL2ndgsZkD5T9UygG0frJ#:uid=322452000635593417592937&h2=The-JSX-Representation-of-a-sc

This is what we have today:
```javascript
export var storyboard = (
  <Storyboard layout={{ layoutSystem: 'pinSystem' }}>
    <Scene
      component={App}
      props={{ myProperty1: 'hello', children: (<div>I am a child prop of app</div>) }}
      style={{ position: 'absolute', left: 0, top: 0, width: 375, height: 812 }}
    />
  </Storyboard>
)
```

This is the proposal from the paper doc (I copied it here in case we loose it for some reason, but please check the paper doc first)
```javascript
export var storyboard = (
  <Storyboard layout={{ layoutSystem: 'pinSystem' }}>
    <Scene
      style={{ position: 'absolute', left: 0, top: 0, width: 375, height: 812 }}
    >
      <App myProperty1="hello"><div>I am a child prop of app</div></App>
    </Scene>
  </Storyboard>
)
```

Notice how `App` is the child of `Scene`, and app has properties on it instead of the scene having a magic passdown property. This makes it **way more** straightforward to work with components that need children as input. (ie a wrapper, or a dynamically sized button etc)

It also removes one more magic from the system, allowing Scene to be a single-purpose component (act as a root for the DOM walker and Spy)

**Comment by maltenuhn** [comment]
I think this is now unblocked and ready to be worked on

**Comment by Rheeseyb** [comment]
Ha! Fixed by #1083 :smile:
